debian/DEBIAN/control

```diff
@@ -0,0 +1,8 @@
+Package: nextcloud
+Version: 12.0.3
+Section: misc
+Priority: optional
+Architecture: amd64
+Depends: php7.0,php7.0-xml,php7.0-cgi,php-cli,php7.0-mysql,php7.0-mbstring,php7.0-gd,php7.0-curl,php7.0-zip,apache2,libapache2-mod-php7.0
```

Comment:
This shouldn't depend on `apache2` imho and just optionally provide integration. It should provide integration with `nginx` as well.

Reply:
i think i split the package into at least 3 peaces 
1. nexcloud
2. nextcloud-apache
3. nextcloud-nginx
this way other Webserver can be integrated as well